public/users/Ayush7614/info.yml

```diff
@@ -0,0 +1,18 @@
+name: Ayush Kumar 
+college: ABES ENGINEERING COLLEGE GHAZIABAD UP INDIA
+location: Ghaziabad, India 
+country: India  
+email: ayushknj3@gmail.com@gmail.com
```

Comment:
I think you accidentally entered an incorrect e-mail address here (I see `@gmail.com` twice. Please fix that. :)